diff -durN qcad-1.4.4-src.orig/rconfig.cpp qcad-1.4.4-src/rconfig.cpp
--- qcad-1.4.4-src.orig/rconfig.cpp	Fri Aug 11 20:31:54 2000
+++ qcad-1.4.4-src/rconfig.cpp	Fri Sep 15 13:23:12 2000
@@ -178,7 +178,7 @@
   addSetting( "Print:PrintAdjustmentFactor", "1.0");
 
   addSetting( "Paths:BrowserPath", (RUNNING_WINDOWS ? "C:\\Progra~1\\Intern~1\\Iexplore.exe" : "netscape"));
-  addSetting( "Paths:LibraryPath1", "/usr/share/qcad/libraries" );
+  addSetting( "Paths:LibraryPath1", DATADIR "/" DEF_APPNAME_L "/libraries" );
 
   verbose=false;
 }
@@ -579,7 +579,7 @@
 
   // Search in /usr/share/qcad/... dir:
   if(!QFileInfo(path).exists()) {
-    path = QString("/usr/share/")+DEF_APPNAME_L+"/"+_relPath;
+    path = QString(DATADIR)+DEF_APPNAME_L+"/"+_relPath;
   }
 #endif
 
diff -durN qcad-1.4.4-src.orig/rfile.cpp qcad-1.4.4-src/rfile.cpp
--- qcad-1.4.4-src.orig/rfile.cpp	Fri Aug 11 20:31:54 2000
+++ qcad-1.4.4-src/rfile.cpp	Fri Sep 15 13:24:02 2000
@@ -295,7 +295,7 @@
 
 #ifndef DEF_WINDOWS
   // Test /usr/share/<proj>/...:
-  ret = QString("/usr/share/")+DEF_APPNAME_L+subdir+"/"+_name;
+  ret = QString(DATADIR)+DEF_APPNAME_L+subdir+"/"+_name;
   RLOG("\nret2: "); RLOG(ret);
   if(QFileInfo(ret).exists()) return ret;
 
@@ -327,7 +327,7 @@
 #ifndef DEF_WINDOWS
   if(_prior==0 || _prior==2) {
     // Test /usr/share/<proj>/...:
-    ret = QString("/usr/share/")+DEF_APPNAME_L+"/"+_subdir;
+    ret = QString(DATADIR)+DEF_APPNAME_L+"/"+_subdir;
     if(QFileInfo(ret).exists()) return ret;
   }
 
diff -durN qcad-1.4.4-src.orig/rfonts.cpp qcad-1.4.4-src/rfonts.cpp
--- qcad-1.4.4-src.orig/rfonts.cpp	Fri Aug 11 20:31:54 2000
+++ qcad-1.4.4-src/rfonts.cpp	Fri Sep 15 13:24:32 2000
@@ -68,10 +68,10 @@
 
 #ifndef DEF_WINDOWS
   // Lookup from /usr/share/<proj>/fonts:
-  lookupFonts( QString("/usr/share/")+DEF_APPNAME_L+"/fonts" );
+  lookupFonts( QString(DATADIR)+DEF_APPNAME_L+"/fonts" );
 
   // Lookup from /usr/local/<proj>/fonts:
-  lookupFonts( QString("/usr/local/")+DEF_APPNAME_L+"/fonts" );
+  lookupFonts( QString(DATADIR)+DEF_APPNAME_L+"/fonts" );
 
   // Lookup from ~/.<proj>/fonts:
   lookupFonts( RCONFIG->getConfigDir().path()+"/fonts" );
