--- qcad-1.5.4/main.cpp.wiget	Thu Apr 24 17:55:23 2003
+++ qcad-1.5.4/main.cpp	Thu Apr 24 17:55:59 2003
@@ -12,9 +12,7 @@
 #include <qfileinfo.h>
 #include <qmessagebox.h>
 #include <qpixmap.h>
-#include <qplatinumstyle.h>
 #include <qtimer.h>
-#include <qwindowsstyle.h>
 
 #include "rappwin.h"
 #include "rconfig.h"
@@ -59,7 +57,7 @@
   
   // Set Application style:
   //
-  QApplication::setStyle( new QWindowsStyle() );
+  QApplication::setStyle( "Windows" );
   //QApplication::setStyle( new QPlatinumStyle() );
   app.setMainWidget(appWin);
 
